File: BackEnd/ajax/ConsultaExterna.js
$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'idMedico': null,
    'idConsultorio': null,
    'diagnostico': null,
    'sintomas': null,
    'idExpediente': null,
    'observacion': null,
    'fechaHora': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorHoy',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroMedicoFecha',
    'idCentroMedico': null,
    'fechaHora': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorConsultorio',
    'idConsultorio': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorMedico',
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idMedico': null,
    'idConsultorio': null,
    'diagnostico': null,
    'idConsulta': null,
    'sintomas': null,
    'idExpediente': null,
    'observacion': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroMedico',
    'idCentroMedico': null,
    'nombreCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarDiariasPorConsultorio',
    'idConsultorio': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/ConsultaExterna.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listar',
    'idConsulta': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/ConsultaExterna.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='ConsultaExterna'
data={
  'accion':'crear',
  'idMedico': None,
  'idConsultorio': None,
  'diagnostico': None,
  'sintomas': None,
  'idExpediente': None,
  'observacion': None,
  'fechaHora': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listarPorPaciente',
  'idExpediente': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listarPorHoy',
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listarPorCentroMedicoFecha',
  'idCentroMedico': None,
  'fechaHora': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listarPorConsultorio',
  'idConsultorio': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listarPorMedico',
  'idMedico': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'actualizar',
  'idMedico': None,
  'idConsultorio': None,
  'diagnostico': None,
  'idConsulta': None,
  'sintomas': None,
  'idExpediente': None,
  'observacion': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listarPorCentroMedico',
  'idCentroMedico': None,
  'nombreCentro': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listarDiariasPorConsultorio',
  'idConsultorio': None,
}
print postear(data,url)
print

url='ConsultaExterna'
data={
  'accion':'listar',
  'idConsulta': None,
}
print postear(data,url)
print
